extensions/JeremyGamer13/tween.js

```diff
@@ -81,224 +128,295 @@
             const tweened = this[easeMethod](progress, easeDirection);
             return this.multiplierToNormalNumber(tweened, start, end);
         }
+        tweenPlayAnimationPosition(args, util) {

```

Comment:
For some reason, the sprite doesn't glide from point A to point B properly when using this block. Try it for yourself - setting mode to "linear", it should glide to the ending point at a fixed rate, but it instead eases in and eases out, and seems to stop before the block finishes.

This is probably because some starting position constant is constantly getting updated with the sprite's current x position whenever it moves forward a step, but it should stay the same throughout the whole animation. Haven't studied where that happens, though.

This does not happen with any of the other blocks.